engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/content-view-composite-content-views-list.controller.js

```diff
@@ -33,6 +33,13 @@ angular.module('Bastion.content-views').controller('ContentViewCompositeContentV
             return deferred.promise;
         };
 
+        $scope.newerVersionExists = function (contentViewVersion) {
+            var found = _.find(contentViewVersion['content_view']['content_view_version_ids'], function(num) {
```

Comment:
With incremental update, the idea of a 'newer' version is somewhat muddled.  Is it by version #? or just last published?  

I would think it'd be by version number (i.e. 3.0 is newer than 2.2) even if the greater number was published before the lesser numbered version.